includes.container/usr/sbin/init.new

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+scriptsPath="/usr/share/init.d"
+nextInitFile="/usr/share/init/nextInit.conf"
+nextInit="/lib/systemd/systemd"
+
+log() {
+  local level="$1"
+  local message="$2"
+  echo "[init] ($level): $message"
+  echo "[init] ($level): $message" > /dev/kmsg
+}
+
+failed() {
+  if [[ -x "/usr/bin/plymouth" ]]; then
+    /usr/bin/plymouth --quit || true
+  fi
+
+  dmesg || true
+  echo "Failed to boot"
+  read -p "Press Enter to shut down"
```

Comment:
I have an idea. What about using the plymouth for this behavior?
Here the [manpage](https://www.unix.com/man-page/linux/1/plymouth/):

```
       ask-question
	   Ask the user a question.

	   --command=STRING
	       Command to send the answer to via standard input

	   --prompt=STRING
	       Message to display when asking the question

	   --dont-pause-progress
	       Don't pause boot progress bar while asking
```

we could check if the plymouth process is still up and use the `ask-question` to show it nicely, then fallback to the CLI if plymouth is not reachable.

Reply:
I considered that but I want the user to see the dmesg output. 
If this happens to someone then either: 
- Our image is broken and we will either see it ourselves in early testing or someone will notice it on their machine and can send us the log
- Or the user is using a custom image and wants to see the log anyway to debug their image

Since the behavior of the init system should always be pretty reliable almost no user will ever see this (hopefully) so the screen doesn't have to be pretty.

Reply:
fair enough